js/app/modules/card/title.js

```diff
@@ -41,6 +42,22 @@ const Title = new Module.Class({
             'Decorate on highlight', 'Whether to draw a custom decoration when the card is highlighted',
             GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT_ONLY,
             false),
+
+        /*
+         * Propety: lines
```

Comment:
@ptomato And to your overall comment, yes, I tested all other templates to check, and the default value of 1 line is working as intended